client/src/ui/ToasterWebview.ts

```diff
@@ -0,0 +1,117 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2023 Savoir-faire Linux. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode'
+
+const DEFAULT_TOASTER_PORT = 8000
+export class ToasterPanel {
+  /**
+  * Track the currently panel. Only allow a single panel to exist at a time.
+  */
+  public static currentPanel: ToasterPanel | undefined
+
+  public static readonly viewType = 'yoctoToaster'
+
+  private readonly _panel: vscode.WebviewPanel
+  private readonly _disposables: vscode.Disposable[] = []
```

Comment:
nothing ever stored in it?

Reply:
For now, there is nothing to store. It depends on what else we want to do with the webview.